libr/core/project.c

```diff
@@ -159,6 +159,56 @@ R_API int r_core_project_delete(RCore *core, const char *prjfile) {
 	return 0;
 }
 
+R_API bool r_core_rop_load(RCore *core, const char *prjfile) {
+	char *path, *db;
+	ut8 found = 0;
+	SdbListIter *it;
+	SdbNs *ns;
+
+	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
+
+	if (!prjfile || !*prjfile) {
+		return false;
+	}
+
+	if (prjfile[0] == '/') {
+		db = r_str_newf ("%s.d", prjfile);
+		if (!db) return false;
+		path = strdup (db);
+	} else {
+		db = r_str_newf (R2_HOMEDIR"/projects/%s.d", prjfile);
+		if (!db) return false;
+		path = r_str_home (db);
+	}
+
+	if (!path) {
+		free (db);
+		return false;
+	}
+
+	if(rop_db) {
+		ls_foreach (core->sdb->ns, it, ns){
+			if (ns->sdb == rop_db){
+				ls_delete (core->sdb->ns, it);
+				found = 1;
+				break;
+			}
+		}
+	}
+	if (!found) sdb_free (rop_db);
+	rop_db = sdb_new (path, "rop", 0);
+	if (!rop_db) {
+		free (db);
+		free (path);
+		return false;
+	}
+	sdb_ns_set (core->sdb, "rop", rop_db);
+	free (path);
+	free (db);
+	return true;
+}
+
```

Comment:
it is removed, initially there were 2 lines before next function, now there is only 1 but the diff doesn't notice this